pyproject.toml

```diff
@@ -196,6 +196,8 @@ ignore_missing_imports = true
 
 [tool.pytest.ini_options]
 testpaths = ["src", "tests"]
+asyncio_mode = "auto"
```

Comment:
I presume this is needed too as the mode was missing?

Reply:
Oh, no, wait, it was already in the templates, I guess only this project was missing it.

Reply:
I actually removed all the  `pytest-asyncio` config from `repo-config` itself because we are not actually using it.

Reply:
ok, I understand. Thanks